### ISSUE TITLE
Checking mail notifcation stays forever if server is not reachabel

### ISSUE THREAD

**Issue by derpeter** [thunderbird/thunderbird-android#770]
It happens from time to time that k9 mail is not able to reach the server and keeps the checking mail notification for ever. I observed this i.e. in wifi networks that have no uplink. Sometimes when i force k9 mail to pull mails in such a network (of course by mistake) that the "cheking mail" notification pops up and stays until i reboot the phone. Killing k9mail via the app settings dialog of android is not helping. It also stays there after k9 successfully pulled mails trough a different wifi network.

I'm running k9 on an nexus 5 with android 5.1.1 the mail server on the other side is a dovecot imap on debian old stable. Latest k9 version tested is 5.006 from fdroid.

**Comment by cketti** [comment]
I have no idea what could be causing this. But most of the notification code was rewritten in PR #795 and it's likely this problem was also fixed by the rewrite.
I will close this issue once the pull request is merged into `master`. If you're still seeing the issue with the next stable release (5.2xx, not yet released, no ETA) please open another issue.
